test/sdil/utils/FakeApplicationSpec.scala

```diff
@@ -225,6 +225,15 @@ trait FakeApplicationSpec extends PlaySpec with BaseOneAppPerSuite with FakeAppl
     m
   }
 
+  lazy val mockPayApiConnector: PayApiConnector = {
```

Comment:
Can we delete this if is not needed or just delete comments

Reply:
Done